lichess-bot.py

```diff
@@ -510,6 +511,7 @@ def game_error_handler(error: BaseException) -> None:
     pool.apply_async(play_game,
                      kwds=play_game_args,
                      error_callback=game_error_handler)
+    # play_game(**play_game_args)
```

Comment:
this seems important - why was it commented?

Reply:
Originally, the line 516 was not there. It was to stop running multiple processes and just run one game at a time. The pool.apply_sync was the original run for play_game that runs multiple games at the same time.

Reply:
OK, this is fine then